python/pyspark/mllib/tests.py

```diff
@@ -75,22 +75,23 @@
     # No SciPy, but that's okay, we'll skip those tests
     pass
 
-ser = PickleSerializer()
```

Comment:
@JoshRosen From the log the tests are running parallel=4.
```
/home/jenkins/workspace/SparkPullRequestBuilder@2/python/run-tests --modules=pyspark-core,pyspark-sql,pyspark-streaming,pyspark-mllib,pyspark-ml --parallelism=4 
```
I will roll back the ser changes in a few.